Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs

```diff
@@ -417,5 +417,31 @@ public void TestDuplicateFollowingEmail()
             Assert.ThrowsAny<ArgumentException>(() => profile.AddFollow(duplicateFollowingEmail));
         }
 
+        [Fact]
+        public void TestRemoveFollowingEmail()
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followingEmail = "testermail@gmail.com";
+            profile.AddFollow(followingEmail);
+
+            // act
+            profile.RemoveFollowing(followingEmail);
+
+            // assert
+            Assert.DoesNotContain(followingEmail, profile.FollowingEmails);
+        }
+
+        [Fact]
+        public void TestRemoveFollowingEmailThrowsExceptionWhenEmailDoesNotExist()
+        {
+            // arrange
+            DomainProfile profile = new("emailtest@gmail.com", "Bob", "Fields");
+            string followingEmail = "testermail@gmail.com";
+
+            // assert
+            Assert.Throws<ArgumentException>(() => profile.RemoveFollowing(followingEmail));
```

Comment:
use ThrowsAny instead of Throws, and IsAssignableTo instead of IsType, pretty much universally